tests/test_cli.py

```diff
@@ -133,23 +127,21 @@ def test_data_file_parsed_by_question_type(
     tmp_path_factory: pytest.TempPathFactory,
 ) -> None:
     src, dst, local = map(tmp_path_factory.mktemp, ("src", "dst", "local"))
-    build_file_tree(
-        {
-            (src / "copier.yml"): (
-                """\
+    build_file_tree({
```

Comment:
Is this a manual change too? I would find it surprising if Black/Ruff concatenated again `({` together instead of adding newlines and indentation levels.

Reply:
both black and ruff do this in --preview mode. likely to move into non-preview eventually but black doesn't move fast

Reply:
Interesting, OK, thanks :slightly_smiling_face:

Reply:
Just to be clear: did you manually run Ruff with preview mode enabled? If yes: is the preview mode enabled in our CI (pre-commit / poe lint / GitHub workflows / etc.)?

If it is not enabled, then I think we should revert all these style changes, and let Ruff format the code again later when such formatting is out of preview mode.

Otherwise, we could consider enabling preview mode, but I'll leave that decision to other @copier-org/maintainers.

Reply:
Locally, I reverted the changes in tests, ran `poe lint` again, and it didn't style the code as what we see in this PR.

Reply:
What happens if you

1. Clone this PR
2. Run 'poe lint' without reverting any other changes
3. Commit and push back to this PR

?

Reply:
Can do. Will need to find some time to manually revert- hopefully tonight